### PR TITLE
demisto-sdk docker readme

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready

Readme for issue: https://github.com/demisto/etc/issues/45932

### CODE REVIEW

File: docs/concepts/demisto-sdk-docker.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+```sh
+alias demisto-sdk="docker run -it --rm \
+--mount type=bind,source="$(pwd)",target=/content \
+--mount source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind \
+demisto/demisto-sdk:<tag>"
+```
+
+### Validate command
+
+for more information about the validate command, please refer to its [documentation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/README.md) on the [demisto-sdk repo](https://github.com/demisto/demisto-sdk).
```
Comment (amshamah419) [inline docs/concepts/demisto-sdk-docker.md:87]:
```suggestion
For more information about the validate command, please refer to its [documentation.](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/README.md) on the [demisto-sdk repo](https://github.com/demisto/demisto-sdk).
```

**Review by amshamah419 — APPROVED** [review]
lgtm

**Comment by glicht** [comment]
This need to go in the following article: https://xsoar.pan.dev/docs/concepts/demisto-sdk

**Review by glicht — DISMISSED** [review]
Please add to: https://xsoar.pan.dev/docs/concepts/demisto-sdk

**Pull request merged**